src/client.ts

```diff
@@ -1365,7 +990,7 @@ export class Client {
     while (off + 4 < decResLengths.getAddresses) {
       if (arePubkeys) {
         // Pubkeys are shorter and are returned as buffers
-        const pubBytes = addrData.slice(off, off + 65);
+        const pubBytes = decrypted.slice(off, off + 65);
```

Comment:
`_handleEncResponse()` used to return `{ err: any, data: any }`, but now it just returns  `any`. This is because with async/await, it's a lot easier to do error handling. So we don't have to pass around `{err, data}` objects, we can just pass around the data and throw errors if things go wrong.